sw/host/opentitanlib/src/app/mod.rs

```diff
@@ -222,6 +222,17 @@ impl TransportWrapperBuilder {
     }
 
     pub fn add_configuration_file(&mut self, file: config::ConfigurationFile) -> Result<()> {
+        if let Some(interface) = file.interface {
+            if self.interface == "" {
+                self.interface = interface;
+            } else if self.interface == interface {
+            } else {
```

Comment:
If this is an intentional do-nothing case, please add a comment.

Reply:
Done.